xml/System.Net.Security/SslCertificateTrust.xml

```diff
@@ -43,10 +43,10 @@
         <Parameter Name="sendTrustInHandshake" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="trustList">To be added.</param>
+        <param name="trustList">The store containig the trusted certificates.</param>
         <param name="sendTrustInHandshake">To be added.</param>
```

Comment:
@wfurt can you please provide description for this param?

Reply:
If the is set to true server  will send list of trusted CAs during TLS Handshake. This is used to guide client to select appropriate client certificate. 

Now, this has many caveats and we should put some big warnings about it: 1. In 6,0 it works only on Windows - we cut out macOS and Linux. 2) on Windows it depends on registry setting. 

Since this increases size of the handshake message and can be viewed as info leak about system configuration, this should be off unless there is specific reason. e.g. it would be nice to have some warning like "don't use it unless you know what you are doing".

Reply:
@wfurt would you mind providing these comments as suggestions? Both for the empty `<param>` and for the `<remarks>` section below with all those caveats you mentioned.

Reply:
I already did that in my review.